clients/daco-dialect-support/syntaxes/daco.injection.json

```diff
@@ -1,6 +1,6 @@
 {
     "scopeName": "daco.injection",
-    "injectionSelector": "L:source.cobol",
+    "injectionSelector": "R:source.cobol",
```

Comment:
change order of injection. Apply DaCo coloring after applying the cobol coloring as the "EXEC" keyword match present in DaCo grammar disrupts the exec block (sql/cics) coloring